backend/capellacollab/sessions/operators/k8s.py

```diff
@@ -177,6 +179,7 @@ def start_persistent_jupyter_session(
             "JUPYTER_BASE_URL": path,
             "JUPYTER_TOKEN": token,
             "JUPYTER_PORT": "8888",
+            "HOST": f"{general_conf.get('scheme')}://{general_conf.get('host')}:{general_conf.get('port')}",
```

Comment:
I would use a different name than `HOST`.

Normally the `HOST` environment variable contains the machine/container host name (no scheme and port).

Reply:
What about `ORIGIN_HOST`? It has to be changed in the Jupyter image of capella-dockerimages repository too.

Reply:
`ORIGIN_HOST` sounds fine. Or something like `CSRF_ORIGIN_HOST`?

Reply:
Renamed to `CSRF_ORIGIN_HOST`.